rpmbuild/SPECS/rocket-depot-py3.spec

```diff
@@ -0,0 +1,46 @@
+Name: rocket-depot
+Version: 849654dd321f070c92a569e26aa0d8a5e114eeb1
```

Comment:
The version needs to be an incremental value, whereas this appears to be potentially a commit hash. Under that assumption, the problem with commit hashes is that a newer one can sort smaller than an older one.

Reply:
You are right; ideally there would be a tag (> 1.0.0) in the main branch , so I can refer to it. Or maybe just call it "1.0.0-py3" abritrarily ?